.speakeasy/gen.yaml

```diff
@@ -1,21 +1,23 @@
-configVersion: 1.0.0
+configVersion: 2.0.0
 generation:
-  comments: {}
```

Comment:
Would this be a breaking change? We may want to bump version with a breaking release if so